packages/e2e-test-utils/src/delete-theme.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Internal dependencies
+ */
+import { activateTheme } from './activate-theme';
+import { installTheme } from './install-theme';
+import { switchUserToAdmin } from './switch-user-to-admin';
+import { switchUserToTest } from './switch-user-to-test';
+import { visitAdminPage } from './visit-admin-page';
+import { isThemeInstalled } from './theme-installed';
+
+/**
+ * Deletes a theme from the site, activating another theme if necessary.
+ *
+ * @param {string} slug        Theme slug.
+ * @param {string?} newThemeSlug A theme to switch to if the theme to delete is active. Required if the theme to delete is active.
+ * @param {string?} newThemeSearchTerm A search term to use if the new theme is not findable by its slug.
+ */
+export async function deleteTheme( slug, newThemeSlug, newThemeSearchTerm ) {
```

Comment:
How do you feel about updating the function signature to put all optional params inside a single param? 
```suggestion
export async function deleteTheme( slug, { newThemeSlug, newThemeSearchTerm } ) {
```